src/mame/video/namcofl.cpp

```diff
@@ -60,19 +60,29 @@ void namcofl_state::RozCB(uint16_t code, int *tile, int *mask, int which)
 
 uint32_t namcofl_state::screen_update_namcofl(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect)
 {
+	/* compute window for custom screen blanking */
+	rectangle clip;
+	//004c 016b 0021 0101 004a 0060 (finalapr*)
+	//004b 016b 0021 0101 0144 0047 (speedrcr)
+	clip.min_x = m_c116->get_reg(0) - 0x4b;
```

Comment:
FYI these can be set with clip.set(minx, maxx, miny, maxy), or even while declaring, like: rectangle clip(minx, maxx, miny, maxy)

Reply:
Additionally this stuff should be internalized and optimized from c116 itself, we currently have 5 different implementations of the same identical code.